kernel/src/path.rs

```diff
@@ -88,7 +88,7 @@ impl<Location: AsUrl> ParsedLogPath<Location> {
         let filename = url
             .path_segments()
             .ok_or_else(|| Error::invalid_log_path(url))?
-            .last()
+            .next_back()
```

Comment:
just optimization to not traverse the entire iterator?

Reply:
Yes, clippy warning fix